packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -85,10 +108,23 @@ export interface IHierBlock extends IMergeBlock {
     rangeStacks: RangeStackMap;
 }
 
+/**
+ * Contains removal information associated to an ISegment.
+ */
 export interface IRemovalInfo {
+    /**
+     * Seq at which this segment was removed.
```

Comment:
We should either define what "seq" means here, or not abbreviate it.

Reply:
I see this "Seq" is used in a number of places. It would be nice to have a common definition we could reference. Still might be a good idea to define the abbreviation in places it's used though.

Reply:
I added a note explicitly mentioning the abbreviation to the README that explains high-level overview of merge-tree.